### ISSUE TITLE
Add Option to change MM/DD/YY to other variation (DD/MM/YY)

### ISSUE THREAD

**Issue by cannabanana** [ArkEcosystem/desktop-wallet#450]
Possible? Doable? Please?

Add Option to change MM/DD/YY to other variation (DD/MM/YY) please, make it so.

**Comment by j-a-m-l** [comment]
Related to https://github.com/ArkEcosystem/ark-desktop/issues/392

**Comment by fluorine** [comment]
Could we use the OS' culture?

**Comment by fluorine** [comment]
I'm almost done with this. I've added the option to the menu (it's pushed in my fork), but I need to implement the usage of the date before a pull request. I cannot see the date in the table without any transaction, and I'm new on this.

How can I get test coins to check the date in the table? I generated this account, with this test address DK5TBbiCokyvFNspWDRx6w5bZrgHywyCB3, can someone send me test arks? Is there an even easier way? Thanks a lot.
